BiconeEvolution/current_antenna_evo_build/XF_Loop/Evolutionary_Loop/Antenna_Performance_Metric/FScorePlotPUEO.py

```diff
@@ -150,25 +141,39 @@
 plt.figure(figsize=(10, 8))
 
 plt.style.use('seaborn')
-#plt.axhline(y=Veff_ARA, linestyle = '--', color = 'k')
 
 colors = cm.rainbow(np.linspace(0, 1, g.NPOP))
 
 # Testing our colorblind friendly colors
 colors2 = ['#00429d', '#3e67ae', '#618fbf', '#85b7ce', '#b1dfdb', '#ffcab9', '#fd9291', '#e75d6f', '#c52a52', '#93003a']
-plt.axis([-1, g.numGens+1, minFit - maxError*1.2 , maxFit + maxError*1.2])
-plt.plot(genAxis, MeanFitness, linestyle='-', alpha = 1, markersize = 15)
-plt.plot(genAxis, MedianFitness, linestyle='dashed', alpha = 1, markersize = 15)
+
+# Scale to include the toyon data
+if veff_toyon > ( maxFit + (diffFit * 0.25)):
+    max_y = veff_toyon  + (diffFit * 0.25)
+else:
+    max_y = maxFit + (diffFit * 0.25)
+
+plt.axis([-1, g.numGens+1, minFit - (diffFit * 0.25) , max_y])
+plt.plot(genAxis, MeanFitness, linestyle='dotted', alpha = 1, markersize = 15, zorder=9)
+plt.plot(genAxis, MedianFitness, linestyle='dashed', alpha = 1, markersize = 15, zorder=9)
+
+# Plotting the Toyon Data
+plt.axhline(y=veff_toyon, color='black', linestyle='solid', alpha = 1, markersize = 15, zorder=9)
+# Plotting the Toyon Error Bars
+plt.axhline(y=veff_toyon + veff_toyon_err_plus, color='gray', linestyle='dotted', alpha = 1, markersize = 15, zorder=9)
+plt.axhline(y=veff_toyon - veff_toyon_err_minus, color='gray', linestyle='dotted', alpha = 1, markersize = 15, zorder=9)
```

Comment:
```suggestion
plt.axhline(y=veff_toyon + veff_toyon_err_plus, color='gray', linestyle='dotted', alpha=1, markersize=15, zorder=9)
plt.axhline(y=veff_toyon - veff_toyon_err_minus, color='gray', linestyle='dotted', alpha=1, markersize=15, zorder=9)
```

Since these are all the same copy-pasted args (with the same extra spaces), consider doing something like:
```suggestion
styles = dict(alpha=1, markersize=15, zorder=9)
plt.axhline(y=veff_toyon + veff_toyon_err_plus, color='gray', linestyle='dotted', **styles)
plt.axhline(y=veff_toyon - veff_toyon_err_minus, color='gray', linestyle='dotted', **styles)
```
to avoid duplication and make the code more readable and easier to read (you can do this at line 155 and reuse it 5 times).